### PR TITLE
Implemented the following functions

### PR DESCRIPTION
**Pull request by csKyou**
・Ask for read permission
・Grant read permission

### CODE REVIEW

File: src/html/js/app.js [hunk 1]
```diff
@@ -250,6 +284,177 @@ createInfoTd = function(i, accountInfo) {
...
+    }).fail(function(error) {
+        console.log(error);
+    });
+}
+
+/*
+ * Get Transcell Token of the external Cell and prepare its data.
+ * When done, execute callback (add external Cell to proper list).
+ */
+Common.prepareExtCellForApp = function(extUrl, profObj, no) {
```
Comment (dixonsiu) [inline src/html/js/app.js]: Please don't overwrite existing method.
Similar method is already defined in common_personium.js.

File: src/html/app.html [hunk 2]
```diff
@@ -61,32 +68,47 @@
                     </li>
                 </ul>
             </div>
+            <div id="other_list">
+                <ul>
+                    <li class="menu-list menu-list-header">
+                        <span>別のCalendarを表示</span>
```
[the remark this answers is not in the document]
Comment (csKyou) [reply src/html/app.html]: i18next correspondence was done

File: src/html/app.html [hunk 3]
```diff
@@ -61,32 +68,47 @@
...
+        </div>
+      </div>
       <div id="dispMsg" style="display:none;position:absolute;background-color: rgba(255,255,255,0.8);width: 100%;z-index:9;"></div>
-      <div id="container" style="height:100%;">
+      <div id="container" style="height:92%;">
         <div class='panel list-group toggle-panel' id='toggle-panel1' style="display:none;"></div>
         <div id='calendar'></div>
       </div>
-      <div id="schedule-scroller" class="fc fc-unthemed fc-ltr" style="height: 100%;overflow:auto;">
+      <div id="schedule-scroller" class="fc fc-unthemed fc-ltr" style="height: 94%;overflow:auto;">
```
[the remark this answers is not in the document]
Comment (csKyou) [reply src/html/app.html]: There is no problem as the height of the header does not change in Japanese and English

File: src/html/js/app.js (hunk 1 above)
Comment (csKyou) [reply src/html/js/app.js]: It has been deleted

Comment (dixonsiu) [reply src/html/js/app.js]: Did you modify anything to fit your purpose?

Comment (csKyou) [reply src/html/js/app.js]: We are also updating Common.prepareExtCellForApp of common_personium.js

**Pull request merged**